packages/ra-core/src/core/useGetRecordRepresentation.spec.tsx

```diff
@@ -93,4 +93,67 @@ describe('useRecordRepresentation', () => {
         );
         screen.getByText('Hello');
     });
+    it('should return the record name at first', () => {
+        render(
+            <UseRecordRepresentation
+                resource="users"
+                record={{
+                    name: 'Ipsum',
+                    title: 'Lorem',
+                    label: 'lorem-ipsum',
+                    reference: '456',
+                    id: '123',
+                    author: 'John Doe',
+                }}
+            />
+        );
+        screen.getByText('Ipsum');
+    });
+    it('should return the record title at second', () => {
+        render(
+            <UseRecordRepresentation
+                resource="users"
+                record={{
+                    title: 'Lorem',
+                    label: 'lorem-ipsum',
+                    reference: '456',
+                    id: '123',
+                    author: 'John Doe',
+                }}
+            />
+        );
+        screen.getByText('Lorem');
+    });
+    it('should return the record label at second', () => {
+        render(
+            <UseRecordRepresentation
+                resource="users"
+                record={{
+                    label: 'lorem-ipsum',
+                    reference: '456',
+                    id: '123',
+                    author: 'John Doe',
+                }}
+            />
+        );
+        screen.getByText('lorem-ipsum');
+    });
+    it('should return the record reference at third', () => {
+        render(
+            <UseRecordRepresentation
+                resource="users"
+                record={{ reference: '456', id: '123', author: 'John Doe' }}
+            />
+        );
+        screen.getByText('#456');
+    });
+    it('should return the record id by default', () => {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
**praise:** ‏Otherwise nice tests :slightly_smiling_face: